specs/http-api/README.md

```diff
@@ -371,7 +371,7 @@ True
 Submits the Order. Alice wants to accept the Quote and execute the transaction.
 
 ### Endpoint
-`POST /exchanges/:exchange_id/order`
+`POST /exchanges/order`
```

Comment:
given that `POST /exchanges` creates a _new_ exchange using an RFQ _without_ the path pattern followed for subsequent endpoints like this one, would it make sense to use `PUT` instead of `POST`?  something feels slightly awkward about this but maybe i'm the only one feeling that way in which case ignore.

Reply:
Agreed, `POST` seems correct for both `POST /exchanges` and `POST /exchanges/order`

Reply:
> something feels slightly awkward about the proposed path but maybe i'm the only one feeling that way in which case ignore.

I'm def open to other ideas, or if this proposal just doesn't feel right, it's no biggie if we reject this one.

Reply:
I added some thoughts here https://github.com/TBD54566975/tbdex/issues/225